Fix dangling issues

simonpoole opened the pull request:
This tries to address all dangling loose ends that need to be resolved before going public.

This addresses: https://github.com/tieguy/OSM-collabmark.org/issues/28 https://github.com/tieguy/OSM-collabmark.org/issues/23 https://github.com/tieguy/OSM-collabmark.org/issues/21 https://github.com/tieguy/OSM-collabmark.org/issues/18

simonpoole committed:
Fix dangling issues

tieguy commented on CPM/Collaborative-Mark-Policy-draft.md:
```diff
@@ -120,10 +120,18 @@ You may use the OSM marks on your own website to identify a hyperlink to the OSM
-You may create things with the marks for your own use. These can be t-shirts, caps, desktop wallpapers, and even cakes! But please don’t sell them. 
+You may create things with the marks for your own use. These can be t-shirts, caps, desktop wallpapers, and even cakes! You may also give these items to other people, either for free or for money, as long as you don't do this professionally or ask for more than cost recovery. (Printing a batch of t-shirts for an event is ok; setting up a web shop where people can order t-shirts is not.)
+
+If you want to sell stuff with OSM's brand on it professionally, you may request a licence under Section 4.4. 
```
@simonpoole I'm not sure what difference you intended between this line and the next one?

simonpoole replied:
I have to agree I C&P Frederiks text but it doesn't really work as is. Will change to:

> You may create things with the marks for your own use. These can be t-shirts, caps, desktop wallpapers, and even cakes! You may also give these items to other people, either for free or for not more money than required for cost recovery.
> 
> If you want to sell stuff with OSM's brand on it professionally, you may request a licence under Section 4.4.

simonpoole replied:
Ah there was a C&P error there with that extra sentence that I didn't notice, fixed too.

tieguy commented on CPM/Collaborative-Mark-Policy-draft.md:
```diff
@@ -171,6 +179,9 @@ You need a trademark licence if you want to make merchandise with the OSM marks
 * truthfully advertise to customers how much of the selling price, if any, will be donated to OSM projects.
 
+### 4.5. Fundraising 
+You need a trademark licence if you are are referring to OpenStreetMap in the context of fundraising in any form. 
```
I would say maybe "if you are using or referring to the OpenStreetMap marks to fundraise in any way"? But that's mostly stylistic.

simonpoole replied:
Improving style is OK :-) Will change to your version

simonpoole committed:
Address wording issues raised by Luis

simonpoole committed:
Rework 5.1 to make it more in line with what we probably really want,
which is not to protect the look and feel as such, but to avoid people
setting up look-a-like sites with nefarious intentions.

simonpoole committed:
Remove comma

simonpoole committed:
"open source"

simonpoole committed:
Typo fixes, re-word 5.1 and remove "national" from 5.4

simonpoole committed:
Typo fix

simonpoole committed:
Language and formatting improvement

simonpoole committed:
Updates after discussing feedback from community

simonpoole committed:
Add question mark

simonpoole committed:
Expand OSM to OpenStreetMap

Expand OSM to OpenStreetMap except where the term has been defined in the definitions (for example "OSM marks",

simonpoole committed:
Fix bogus reference from original WMF text

simonpoole committed:
Remove superfluous text

simonpoole committed:
Remove superfluous "to" add SOTM section from wiki

simonpoole committed:
Added plural "s"

simonpoole committed:
Make requirements for selling stuff clearer

simonpoole committed:
Add point to 4.4 avoid "dispel"

simonpoole committed:
Add section on domain names for local user groups

simonpoole committed:
Update Collaborative-Mark-Policy-draft.md

simonpoole committed:
Update Collaborative-Mark-Policy-draft.md

simonpoole committed:
Update Collaborative-Mark-Policy-draft.md

simonpoole committed:
Clean up for publication

simonpoole committed:
Update FAQ.md
